.travis.yml

```diff
@@ -2,6 +2,8 @@ language: node_js
 node_js:
   - 12.4.0
 cache: npm
+before_install:
+  - echo "//registry.npmjs.org/:_authToken=$(npx @manifoldco/cli export -t manifold -p shadowcat-ci | grep NPM_TOKEN | cut -f2 -d "=")" >> ~/.npmrc
```

Comment:
This now uses the `shadowcat-ci` project in GitHub for creds (cred in Travis no longer needed)